docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,827 @@
+---
+title: "Monitor Air Quality with a Fleet of Sensors"
+linkTitle: "Air Quality Fleet"
+type: "docs"
+description: "Use a fleet of machines with air quality sensors to monitor PM 2.5 levels in different indoor and outdoor locations."
+images: ["/tutorials/air-quality-fleet/lower-aqi.png"]
+imageAlt: "A web dashboard showing PM2.5 readings from two air quality sensors."
+tags: ["tutorial"]
+authors: ["Jessamy Taylor"]
+languages: ["typescript"]
+viamresources: ["sensor", "data_manager"]
+level: "Intermediate"
+date: "2024-04-23"
+# updated: ""  # When the tutorial was last entirely checked
+cost: 200
+---
+
+As wildfire smoke, car exhaust, and cooking oils pollute our air, we can become more aware of the pollution levels in the spaces we spend time by collecting data.
+Then, when we take steps to mitigate the problem, the data allows us track whether our interventions are effective.
```

Comment:
Was hoping to provide some reason as to why someone would want to collect this data with a fleet of sensing devices, but I guess since that's not the focus of the tutorial it doesn't matter? Removing

Reply:
As you make the rest of the changes see if you can include that reasoning as part of the story. I’m not opposed to starting with it but it currently feels unnecessary and a bit awkward.